configuration-files/README.md

```diff
@@ -0,0 +1,292 @@
+* [Configuration files](#configuration-files)
+    * [Split by topic](#split-by-topic)
+        * [The base](#the-base)
+        * [API](#api)
+        * [Result](#result)
+    * [Different Access-Control](#different-access-control)
+        * [Base](#base)
+        * [Dockerfile](#dockerfile)
+        * [Stage environment file](#stage-environment-file)
+        * [Production environment file](#production-environment-file)
+
+# Configuration files
+
+Couper gets configured by a [configuration-file](https://github.com/avenga/couper/tree/master/docs#configuration-file) and
+with the [latest 1.9 release](https://github.com/avenga/couper/releases/latest) Couper has an option to handle multiple
```

Comment:
```suggestion
as of the [1.9 release](https://github.com/avenga/couper/releases/tag/v1.9.0) Couper has an option to handle multiple
```

Reply:
Better rephrase: Otherwise we'll have to remember to change that line when releasing 1.10.